agent/config/config_windows.go

```diff
@@ -24,16 +24,40 @@ import (
 
 const (
 	defaultCredentialsAuditLogFile = `log\audit.log`
-	httpPort                       = 80
+	// When using IAM roles for tasks on Windows, the credential proxy consumes port 80
+	httpPort = 80
```

Comment:
nit: lint would dictate that this starts with `httpPort specifies the credentials ...`